internal/pkg/service/buffer/worker/node/config.go

```diff
@@ -0,0 +1,45 @@
+package node
```

Comment:
No a tu mame implementaciu, je to asi 3 pokus.

Fungovali aj tie predchadzajuce pokusy, ale nebol som s nimi spokojny:
- Bud to bolo prilis zlozite.
- Alebo to prilis zatazovalo etcd cluster.

S touto implementaciou som spokojny, vyzaduje minimum komunikacie.

Popis je v nizsie.

Odporucam pozriet najprv testy:
- `TestConsistentHashLib`
- `TestNodesDiscovery`

-----------------

Tato abstrakcia riesi tento problem:
- mam 3 worker nodes
- mam 1000 receivers, ktore treba kazdych 30s kontrolovat
- ako co najjednoduchsie urcim, ktory worker ma riesit, ktore receivers?
- pocet nodes sa moze dynamicky menit:
  - vypadok
  - novy deployment, atd.
  
Pouziva sa `consistent hashing/HashRing` algoritmus/libka:
- https://www.youtube.com/watch?v=UF9Iqmg94tk

---------------

Mozno to vyzera zlozito, ale z testu by to mohlo byt jasne, ono je to vlastne velmi jednoduche:
- libka ma nejakych 250 riadkov `github.com/lafikl/consistent`
- moj kod ma nejakych 250 riadkov
- a etcd watch je tiez jednoduchy na pochopenie

----

Povodne som chcel vyuzit Etcd Election API a organizovat volby pre kazde `receiver-id`.
No zistil som, ze:
- Je to vela komunikacie, ktora rastie s poctom receivers.
- Nevedel som dobre zabezpecit rovnomernu distribuciu medzi workers.
- Ktory worker prvy nastartoval, tak ten vyhral volby pre vsetky receivers.
- Je to vhodne na volbu `leader` v nejakom custom cluster, ale nie na toto.